examples/step-63/step-63.cc

```diff
@@ -1180,8 +1180,9 @@ namespace Step63
   template <int dim>
   void AdvectionProblem<dim>::run()
   {
-    for (unsigned int cycle = 0; cycle < (settings.fe_degree == 1 ? 7 : 5);
-         ++cycle)
+    const unsigned int n_cycles = settings.fe_degree == 1 ? 7 : 5;
+
+    for (unsigned int cycle = 0; cycle < n_cycles; ++cycle)
```

Comment:
Wouldn't
```suggestion
    for (int cycle = 0; cycle < (settings.fe_degree == 1 ? 7 : 5);
         ++cycle)
```
or
```suggestion
    for (unsigned int cycle = 0; cycle < (settings.fe_degree == 1 ? 7u : 5u);
         ++cycle)
```
also work?
Otherwise, I would at least add some parentheses
```suggestion
    const unsigned int n_cycles = (settings.fe_degree == 1 ? 7 : 5);

    for (unsigned int cycle = 0; cycle < n_cycles; ++cycle)
```

Reply:
Thanks! I opted for your second suggestion.